app/app.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
     compile(group = "com.google.code.gson", name = "gson", version = "2.8.1")
     fun testFx(name: String, version: String = "4.0.+") =
         create(group = "org.testfx", name = name, version = version)
+    runtime(group = "edu.wpi.first.ntcore", name = "ntcore-jni", version = "+", classifier = "all")
```

Comment:
What worries me about that is I get it, but our release script on Jenkins depends on plus. We would have to change our entire release cycle just to have this be static. None of our other repos use ststic, and never have.